reana_workflow_controller/config.py

```diff
@@ -59,6 +59,14 @@
 )
 """Serial workflow engine version."""
 
+REANA_KUBERNETES_JOBS_MEMORY_LIMIT = os.getenv("REANA_KUBERNETES_JOBS_MEMORY_LIMIT")
+"""Maximum memory user's Kubernetes jobs can consume.
```

Comment:
Maximum memory limit for user jobs.

(see verbiage suggestion in `reana` repository)